packages/ai/tsup.config.ts

```diff
@@ -62,4 +62,12 @@ export default defineConfig([
       };
     },
   },
+  // MCP stdio
+  {
+    entry: ['mcp-stdio/index.ts'],
+    outDir: 'mcp-stdio/dist',
+    format: ['cjs', 'esm'],
+    dts: true,
+    sourcemap: true,
+  },
```

Comment:
awesome! would be good to try using this in a next application to see if there are any warnings/errors